src/helper_functions.jl

```diff
@@ -645,3 +645,21 @@ A copy of the vector where each negative entry is replaced by zero.
 function rectify(x::AbstractVector{N}) where {N<:Real}
     return map(xi -> max(xi, zero(N)), x)
 end
+
+"""
+    iscounterclockwise(result, correct_expr) where {N<:Real}
+
+Returns a boolean, i.e., take the element-wise maximum with zero.
```

Comment:
I think this description is not informative (it might even be the wrong function?).